src/components/ContentView/ContentView.tsx

```diff
@@ -31,20 +70,29 @@ export const ContentView: React.FC<ContentViewProps> = ({ contentData, pageTitle
     if (!contentData || !contentData[pageString]) {
         return <></>
     }
-
+    console.log("something: " + Math.round(activeSection.index / sections.length * 100.0));
     return <>
-        {/** TODO: Add sidebar here if the page is a sidebar. */}
-        <div id="content-view-container">
-            {contentData[pageString].contentOrder &&
-                contentData[pageString].content &&
-                contentData[pageString].contentOrder!.map((contentHash) => {
-                    let content = contentData[pageString].content![contentHash];
-                    let ContentWidget = ContentMapping[content!.type].widget;
-                    return <div id={contentHash} key={contentHash}>
-                        <ContentWidget {...content} />
-                    </div>
-                })
+        <div className={contentData[pageString].hasSidebar ? "sidebar-content-view" : ""}>
```

Comment:
factor out `contentData[pageString].hasSidebar` since you use it multiple times.